backend/benefit/applications/services/ahjo_authentication.py

```diff
@@ -23,12 +44,12 @@ def __init__(self) -> None:
         self.client_secret: str = settings.AHJO_CLIENT_SECRET
         self.redirect_uri: str = settings.AHJO_REDIRECT_URL
 
-        self.grant_type_for_auth_token: str = "authorization_code"
-        self.grant_type_for_refresh_token: str = "refresh_token"
+        self.grant_type_for_auth_token: str = AUTH_TOKEN_GRANT_TYPE
+        self.grant_type_for_refresh_token: str = REFRESH_TOKEN_GRANT_TYPE
         self.headers: Dict[str, str] = {
             "Content-Type": "application/x-www-form-urlencoded",
         }
-        self.timeout: int = 10
+        self.timeout: int = 60
```

Comment:
Should this be ENV controlled?

Reply:
I'm thinking of refactoring this at some point to  a AhjoClient class which would handle all Ahjo requests as now they are done all over the ahjo_integration.py , so I could do this at that point.